### PR TITLE
website: Add locale to Leaderboard and organize locale files

### PR DESCRIPTION
**Pull request by rjmacarthy**
This is another follow up PR for localization. New locales have been applied to the Leaderboard page and some keys have been re-arranged in order to re-use the common ones.

If you have any questions or concerns please let me know.

### CODE REVIEW

File: website/public/locales/en/leaderboard.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (fozziethebeat) [inline website/public/locales/en/leaderboard.json:1]: Just checking my understanding here.  Does the `leaderboard.tsx` page automatically check this file first for entries?

**Review by fozziethebeat — APPROVED** [review]

**Comment by fozziethebeat** [comment]
We had a merge that affects one of the files you changed.  Please resolve that conflict then I'll merge it.

**Review by fozziethebeat — CHANGES_REQUESTED** [review]
Testing on main I see a lot of issues.  I made a bug that needs to be addressed before this is submitted: https://github.com/LAION-AI/Open-Assistant/issues/870

File: website/public/locales/en/leaderboard.json (hunk 1 above)
Comment (rjmacarthy) [reply website/public/locales/en/leaderboard.json:1]: Hey thanks for the review. To explain a little clearer, to load the translations you have to use the `useTranslation` hook and pass the namespace you wish to load. I've been keeping namespaces per page apart from common ones used for page level components. We could use one huge file but I'm not sure if it's the best way forward. Separate namespaces allow quicker loading times along with maintainable json files. I hope that answers your question?

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**